### DISCUSSION TITLE
Sort Order in Errsole Dashboard should be newest on the top

### DISCUSSION THREAD

**Discussion by dejf-js** [errsole/errsole.js#121] [Ideas]
The order of log records should be set to DESC so that the newest log entries appear at the top, rather than further down. This way, I don’t need to search for the latest entry first to see my new log entry.

Ideally, a switch widget could allow users to change the order of log records according to their preference.

**Comment by mrrishimeena** [comment]
We went with "latest on bottom" since it aligns with the way logs traditionally grow making it easier to follow the flow of events and many tools follow the same approach. That said, adding a switch to let you choose your preferred order sounds like a great idea—we’re on it!

**Reply by dejf-ts** [reply]
I see, it is more naturally to follow the bottom. Maybe it's ok so and just a 'jump to latest records' would do the job. Now I think I have to click "_There may be newer logs to load. Load More_" always I want to see the latest log entries.

**But I see now a new "refresh" feature in the gui !**

Using text logs, there are like nodepad++ plugins that periodicaly jumps automatically to the end of the file. So maybe such a functionality would be enought.

Btw. we started to use Errsole as the single target of logs. Very great is the data retention, filters and the View Meta feature.

**Reply by mrrishimeena** [reply]
Thank you! We are working on enhancing the user experience and adding new features, including an option that allows users to choose their preferred log order.

**Reply by dejf-js** [reply]
Thanks! Found it and working great.
